pkg/ibmcsidriver/ibm_csi_driver_test.go

```diff
@@ -55,6 +65,64 @@ func initIBMCSIDriver(t *testing.T) *IBMCSIDriver {
 	return icDriver
 }
 
+// FakeNodeMounterWithCustomActions ...
```

Comment:
This should maybe move here? https://github.com/IBM/ibm-csi-common/blob/b1ab7008f8e435da0f0f507c4890adc48c805133/pkg/mountmanager/fake-safe-mounter.go#L25